modules/python/test/test_misc.py

```diff
@@ -282,12 +282,11 @@ def test_parse_to_int64_convertible(self):
 
     def test_parse_to_int64_not_convertible(self):
         min_int64, max_int64 = get_limits(ctypes.c_longlong)
-        for not_convertible in (1.2, np.float(4), float(3), np.double(45), 's', 'str',
+        for not_convertible in (1.2, float(3), np.double(45), 's', 'str',
```

Comment:
`np.double`

Reply:
and np.float(4) is float(4) which is already tested as float(3), hence the removal